src/main.rs

```diff
@@ -129,7 +129,7 @@ impl TryFrom<ApduHeader> for Instruction {
 }
 
 #[cfg(any(target_os = "stax", target_os = "flex"))]
-fn show_status_if_needed(ins: &Instruction, tx_ctx: &TxContext, status: &AppSW) {
+fn show_status_if_needed(ins: &Instruction, tx_ctx: &mut TxContext, status: &AppSW) {
```

Comment:
I would rename this one, should be something like `show_home_and_status_if_needed` it's a bit long but it says what it does

Reply:
Then I would recommend `show_status_and_home_if_needed`